demes/load_dump.py

```diff
@@ -106,6 +106,31 @@ def _unstringify_infinities(data: MutableMapping[str, Any]) -> None:
             migration["start_time"] = float(start_time)
 
 
+def _no_null_values(data: MutableMapping[str, Any]) -> None:
+    """
+    Checks for any null values in the input data.
+    """
+
+    def check_if_None(key, val):
+        if val is None:
+            raise ValueError(f"{key} cannot have value `null`")
```

Comment:
Maybe the error could be:
```suggestion
            raise ValueError(f"{key} must have a non-null value")
```
I think this is slightly less confusing in the case where someone accidentally omits a value (which might be more common than deliberately setting a value to `null`?).

Reply:
Good point - I'll change that.